templates/latex/doxygen.sty

```diff
@@ -3,14 +3,14 @@
 
 % Packages used by this style file
 \RequirePackage{alltt}
-\RequirePackage{array}
+%%\RequirePackage{array} %% moved to refman.tex due to workaround for LatEx 2019 version and unmaintained tabu package
 \RequirePackage{calc}
 \RequirePackage{float}
-\RequirePackage{ifthen}
+%%\RequirePackage{ifthen} %% moved to refman.tex due to workaround for LatEx 2019 version and unmaintained tabu package
 \RequirePackage{verbatim}
 \RequirePackage[table]{xcolor}
-\RequirePackage{longtable}
-\RequirePackage{tabu}
+%%\RequirePackage{longtable} %% moved to refman.tex due to workaround for LatEx 2019 version and unmaintained tabu package
```

Comment:
is it really necessary to move these to refman.tex? Can't we just do the \RequirePackage{longtable_doxygen} here instead?

Reply:
Test with doxygen_manual indicates that this is possible, created updated pull request.